reconcile/queries.py

```diff
@@ -338,3 +340,73 @@ def get_users():
     """ Returnes all Users. """
     gqlapi = gql.get_api()
     return gqlapi.query(USERS_QUERY)['users']
+
+
+APP_INTERFACE_SQL_QUERIES_QUERY = """
+{
+  sql_queries: app_interface_sql_queries_v1 {
+    name
+    namespace
+    {
+      name
+      managedTerraformResources
+      terraformResources
+      {
+        provider
+        ... on NamespaceTerraformResourceRDS_v1
+        {
+          identifier
+          output_resource_name
+          defaults
+        }
+      }
+      cluster
+      {
+        name
+        serverUrl
+        automationToken
+        {
+          path
+          field
+          format
+        }
+        internal
+      }
+    }
+    identifier
+    overrides
+    {
+      db_host
+      db_port
+      db_name
+      db_user
+      db_password
+    }
+    query
+  }
+}
+"""
+
+
+def get_app_interface_sql_queries():
+    """ Returns SqlQuery resources defined in app-interface """
+    gqlapi = gql.get_api()
+    return gqlapi.query(APP_INTERFACE_SQL_QUERIES_QUERY)['sql_queries']
+
+
+RESOURCE_QUERY = """
+query Resource($path: String) {
+    resources: resources_v1 (path: $path) {
+        path
+        content
+        sha256sum
+    }
+}
+"""
+
+
+def get_resource(path):
+    gqlapi = gql.get_api()
+    raw_values = gqlapi.query(RESOURCE_QUERY,
+                              {'path': path})['resources'][0]
+    return anymarkup.parse(raw_values['content'], force_types=None)
```

Comment:
why is this needed here?
https://github.com/app-sre/qontract-reconcile/blob/master/utils/gql.py#L56-L79

Reply:
Fixed!